### ISSUE TITLE
Firmwares Not Working - Glitchy, Quiet Output and Encoder Does Nothing

### ISSUE THREAD

**Issue by angiamusic** [pierstu/braids_tht#1]
Hey!

First of all, thank you for putting in the work to compile the hex files for use. I've been having issues trying to get them to work on my THT Braids, though; 

When I flash the firmware onto the bluepill, the module seems to forget the firmware every time I switch it on and starts in the "RDY" mode for wav flashing. If I flash the firmware onto it via wav audio then it goes to being stuck on CSAW with a glitchy, quiet output and forgets the firmware once I turn it off. The encoder seems to do nothing.

I've only been able to get the firmware on the Sound-Force website working with the Braids, nothing else. I tried compiling the firmware on my own after many failed attempts in the Mutable dev environment and it seems to have the same issues as the ones you posted here.

I know this entire project is essentially based on non-profit contributions, but would you happen to have any idea why this could be happening? 

Regards,
Aditya

**Comment by pierstu** [comment]
Yeah i just finished building my copies and actually nothing works :/ totally sorry
I ended up using the Soundforce bootloader + hex combo for reversed rotation encoders 
Should I delete this repo?

**Comment by angiamusic** [comment]
I guess, if it doesn't work then no use :(

@juanitohm on instagram (the M4TM guy) sent me a copy of Renaissance that worked on my THT Braids, maybe get in touch with him and see how to make it all work?
